handlers/photo-audio-haptics-handler/src/server.ts

```diff
@@ -203,24 +220,41 @@ app.post("/handler", async (req, res) => {
                         entities[0] = {
                             ...entities[0],
                             centroid: [[]],
-                            contourPoints: [[]]
+                            contours: [[]],
+                            entityType:"staticSemSeg"
                         };
 
-                    if (preObjDet && preSemSeg)
+                    if (preObjDet && preSemSeg) {
                         entities[1 + segGeometryData.length] = {
                             ...entities[1 + segGeometryData.length],
                             centroid: [[]],
-                            contourPoints: [[]]
+                            contours: [[]],
+                            entityType: "staticObjDet"
```

Comment:
So these are individual names representing an a specific part of TTS? It works, but keep in mind this will make it difficult to scale up without modifying the browser and will heavily limit reusability.